packages/nuxt/src/app/components/nuxt-route-announcer.ts

```diff
@@ -0,0 +1,49 @@
+import { defineComponent, h } from 'vue'
+import type { PolitenessValue } from '#app/composables/route-announcer'
+import { Politeness, useRouteAnnouncer } from '#app/composables/route-announcer'
+
+export default defineComponent({
+  name: 'NuxtRouteAnnouncer',
+  props: {
+    atomic: {
+      type: Boolean,
+      default: false
+    },
+    politeness: {
+      type: String as () => PolitenessValue,
+      default: Politeness.Polite,
+      validator: (value: PolitenessValue) => Object.values(Politeness).includes(value as Politeness)
+    }
+  },
+  setup (props, { slots, expose }) {
+    const { set, polite, assertive, message, politeness } = useRouteAnnouncer({ politeness: props.politeness })
+
+    expose({
+      set, polite, assertive, message, politeness
+    })
+
+    return () => h('nuxt-route-announcer', {
+      class: 'nuxt-route-announcer',
+      style: {
+        position: 'absolute'
+      }
+    }, h('div', {
```

Comment:
do we need to render the `h('nuxt-route-announcer')`? Couldn't we just render the div?

Reply:
Not really, you are right it's not necessary. Will change it in a half an hour.
It is an artifact of my initial implementation with web-component `nuxt-route-announcer`

Reply:
`nuxt-route-announcer` replaced with `div`  ✅